src/modules/rating/rating.spec.ts

```diff
@@ -1,9 +1,14 @@
+///<reference path="../../../typings/jasmine/jasmine.d.ts"/>
+///<reference path="../../../typings/angularjs/angular-mocks.d.ts"/>
+
+import { smRatingModule } from './rating';
+
 describe('Semantic-UI: Components - smRating', function() {
   'use strict';
 
-  var $rootScope, $compile, element;
+  let $rootScope, $compile, element;
 
-  beforeEach(module('semantic.ui.components.rating'));
+  beforeEach(angular.mock.module(smRatingModule.name));
 
   beforeEach(inject(function(_$rootScope_, _$compile_) {
     $rootScope = _$rootScope_.$new();
```

Comment:
Can you change this to use new scope from $rootScope.$new() and use $injector to get $compile, please ?